src/webapp/components/selectmulti-checkboxes/SelectMultiCheckboxes.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ChangeEvent } from "react";
+import Select from "@material-ui/core/Select";
+import Checkbox from "@material-ui/core/Checkbox";
+import MenuItem from "@material-ui/core/MenuItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import InputLabel from "@material-ui/core/InputLabel";
+import FormControl from "@material-ui/core/FormControl";
+import styled from "styled-components";
+
+type Option = {
+    text: string;
+    value: string;
+};
+
+type Props = {
+    label: string;
+    options: Option[];
+    value: string[];
+    onChange: (event: ChangeEvent<any>) => void;
```

Comment:
We try to avoid the `any` type. Since we can select multiple items we can return an array with the selected values.

```tsx
onChange: (values: string[]) => void;
```